### PR TITLE
Settings docs update

### LINKED ISSUE #27949: Update Settings docs
https://docs.mantidproject.org/nightly/workbench/settings.html

there are now more settings (see useful section of Mantid basic tutorial)

### PR DESCRIPTION
**Pull request by DanielMurphy22**
**Description of work.**

**To test:**

Build Docs and check Settings.html makes sense and is formatted correctly:
- Are images the right size? Do sections of text line up with images?
- Does the link work correctly?
- Are explanations clear?

Fixes #27949

*This does not require release notes* because **documentation**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/workbench/settings.rst [hunk 1]
```diff
@@ -4,25 +4,81 @@
...
 
 The settings menu in workbench can be used to change your personal user defaults
 and modify some of the functionality of workbench.
 
 General
 =======
 
-The general tab contains settings covering general user preferences.
 
 - Facility and default Instrument: These settings control the default settings used
```
Comment (ConorMFinn) [inline docs/source/workbench/settings.rst:19]:
```suggestion
- Facility and Default Instrument: These settings control the default settings used
```

File: docs/source/workbench/settings.rst [hunk 2]
```diff
@@ -4,25 +4,81 @@
...
+   :alt: SettingsCategories
+
+|
+|
+|
+
+Categories
+==========
+
+- Choose which Algorithms categories to show in the Algorithm Toolbox.
```
Comment (ConorMFinn) [inline docs/source/workbench/settings.rst:43]:
```suggestion
- Choose which Algorithm categories to show in the Algorithm Toolbox.
```

**Review by cailafinn — CHANGES_REQUESTED** [review]
Just a couple of *incredibly* minor nitpicks, otherwise builds correctly and looks good.

**Review by cailafinn — APPROVED** [review]
Looks good to me now, nice work 👍

**Pull request merged**